src/Compilers/VisualBasic/Test/Syntax/Parser/ParseExpression.vb

```diff
@@ -94,11 +94,28 @@ Public Class ParseExpressionTest
 
 #Region "Literal Test"
 
-
-    <Fact>
-    Public Sub ParseIntegerLiteralTest()
+    <Fact, Trait("IntegerLiteral_Prefix", "Hex")>
+    Private Sub ParseIntegerLiteralTest_HexPrefix()
```

Comment:
Consider making the test methods `Public`, for consistency with other tests, but also because some test runners may ignore `Private` methods.

Reply:
The use of `Public` is mandatory here. When labeled `Private` the tests do not run at all.

Reply:
@cston  will do.